src/Appwrite/Migration/Version/V19.php

```diff
@@ -706,7 +705,7 @@ protected function fixDocument(Document $document): Document
                 }
 
                 $commands = $this->getFunctionCommands($document);
-                $document->setAttribute('commands', $commands);
+                $document->setAttribute('commands', $document->getAttribute('commands', $commands));
```

Comment:
We need to avoid creating a schedule if a schedule already exists 👇🏼

Reply:
added a check for it